code/obj/flock/structure/flock_structure_parent.dm

```diff
@@ -16,6 +16,7 @@
 	var/bruteVuln = 1.2
 	var/fireVuln = 0.2 // very flame-retardant
 	var/datum/flock/flock = null
+	var/passthrough = 0 //can flockdrones pass through this akin to a grille? need to set USE_CANPASS to make this work however
 	var/poweruse = 0 //does this use(/how much) power? (negatives mean it makes power)
 	var/usesgroups = 0 //not everything needs a group so dont check for everysingle god damn structure
```

Comment:
TRUE/FALSE

Reply:
id prefer not to use TRUE/FALSE as all other flock code uses 1/0 for consistencies sake

Reply:
then all other flock code smells, please use TRUE/FALSE for values intended to be boolean

Reply:
i dont like TRUE/FALSE as it doesnt light up in dm,
do you want me to go through all flockcode that exists and use TRUE/FALSE? or just for this pr/future?